From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Sun, 20 Sep 2020 10:25:01 +0900
Subject: [PATCH] Add ENABLE_DEVS: Install libfontforge devels


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 03308e073..ca785ed9f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,6 +43,7 @@ add_uninstall_target()
 
 # Options
 build_option(BUILD_SHARED_LIBS       BOOL ON  "Build libfontforge as a shared library")
+build_option(ENABLE_DEVS             BOOL OFF "Install libfontforge devels: its archive and includes")
 build_option(ENABLE_GUI              BOOL ON  "Build FontForge with GUI support")
 build_option(ENABLE_X11              BOOL OFF "Build the GUI using the X backend INSTEAD of the GDK backend" "ENABLE_GUI")
 
diff --git a/fontforge/CMakeLists.txt b/fontforge/CMakeLists.txt
index 67f91605c..51a19f427 100644
--- a/fontforge/CMakeLists.txt
+++ b/fontforge/CMakeLists.txt
@@ -279,3 +279,8 @@ if(BUILD_SHARED_LIBS)
     install(TARGETS fontforge RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
   endif()
 endif()
+
+if(ENABLE_DEVS)
+  install(TARGETS fontforge ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+  install(FILES ${FONTFORGE_INST_HEADERS} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/fontforge")
+endif()
