From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Sun, 20 Sep 2020 10:25:02 +0900
Subject: [PATCH] libfontforge: Set PROPERTY SOVERSION 4


diff --git a/fontforge/CMakeLists.txt b/fontforge/CMakeLists.txt
index 51a19f427..93714d6cc 100644
--- a/fontforge/CMakeLists.txt
+++ b/fontforge/CMakeLists.txt
@@ -217,6 +217,7 @@ endif()
 set_property(SOURCE splineoverlap.c APPEND PROPERTY COMPILE_OPTIONS ${FONTFORGE_EXTRA_CFLAGS})
 
 set_property(TARGET fontforge PROPERTY VERSION 4)
+set_property(TARGET fontforge PROPERTY SOVERSION 4)
 if(BUILD_SHARED_LIBS)
   if(WIN32 OR CYGWIN)
     target_link_options(fontforge PRIVATE -Wl,--export-all-symbols) # FIXME
