--- origsrc/fontforge-20090923/Makefile.dynamic.in	2008-11-13 23:31:07.000000000 -0600
+++ src/fontforge-20090923/Makefile.dynamic.in	2010-03-02 22:19:02.839034900 -0600
@@ -99,7 +99,7 @@ install_po:
 
 install_pycontrib:
 	mkdir -p $(DESTDIR)$(pythondir)
-	( cd pycontrib ; $(INSTALL_DATA) *.py $(DESTDIR)$(pythondir) )
+	( cd $(srcdir)/pycontrib ; $(INSTALL_DATA) *.py $(DESTDIR)$(pythondir) )
 
 install_prog:
 	mkdir -p $(DESTDIR)$(bindir)
--- origsrc/fontforge-20090923/fontforge/Makefile.dynamic.in	2009-09-23 17:16:48.000000000 -0500
+++ src/fontforge-20090923/fontforge/Makefile.dynamic.in	2010-03-02 22:19:02.844035200 -0600
@@ -76,20 +76,20 @@ DLIBS = -rpath $(libdir)  ../libfontforg
 all: @FINAL_TARGET@ @MACAPP@ sfddiff fontimage fontlint
 
 fontforge: $(@exeOBJECTS@) $(fontforge_LIBOBJECTS) exelibstamp.pre libstamp.pre
-	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < exelibstamp.pre > exelibstamp.c
+	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < $(srcdir)/exelibstamp.pre > exelibstamp.c
 	$(CC) -c $(CFLAGS) exelibstamp.c
-	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < libstamp.pre > libstamp.c
+	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < $(srcdir)/libstamp.pre > libstamp.c
 	$(CC) -c $(CFLAGS) libstamp.c
 	$(LIBTOOL) --mode=link $(CC) -export-dynamic $(LDFLAGS) -o fontforge \
 	    $(@exeOBJECTS@) $(fontforge_LIBOBJECTS) exelibstamp.o libstamp.o $(LIBS)
 
 ../libfontforge.la: $(@exeOBJECTS@)  $(fontforge_LIBOBJECTS) exelibstamp.pre libstamp.pre
-	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < libstamp.pre > libstamp.c
+	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < $(srcdir)/libstamp.pre > libstamp.c
 	$(LIBTOOL) --mode=compile $(CC) $(CFLAGS) -c libstamp.c
 	$(LIBTOOL) --mode=link $(CC) -no-undefined $(CFLAGS) $(LDFLAGS) \
 	    -o ../libfontforge.la $(fontforge_LIBOBJECTS) libstamp.lo $(LIBLIBS) \
 	    -rpath $(libdir) -version-info $(FF_VERSION):$(FF_REVISION):$(FF_AGE)
-	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < exelibstamp.pre > exelibstamp.c
+	sed -e '1,$$ s/REPLACE_ME_WITH_MAJOR_VERSION/$(FF_VERSION)/' -e '1,$$ s/REPLACE_ME_WITH_MINOR_VERSION/$(FF_REVISION)/' < $(srcdir)/exelibstamp.pre > exelibstamp.c
 	$(LIBTOOL) --mode=compile $(CC) $(CFLAGS) -c exelibstamp.c
 	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o fontforge \
 	    $(@exeOBJECTS@) exelibstamp.lo ../libfontforge.la $(LIBS)
