--- origsrc/fontforge-20170731/fontforgeexe/prefs.c	2017-07-30 16:49:01.000000000 -0500
+++ src/fontforge-20170731/fontforgeexe/prefs.c	2018-02-27 16:58:40.756299000 -0600
@@ -595,13 +595,9 @@ static void ProcessFileChooserPrefs(void
 	b[i++] = uc_copy("/Library/Fonts/");
 	b[i++] = uc_copy("/System/Library/Fonts/");
 #endif
-#if __CygWin
-	b[i++] = uc_copy("/cygdrive/c/Windows/Fonts/");
-#endif
-	b[i++] = uc_copy("/usr/X11R6/lib/X11/fonts/");
-#ifndef __CygWin		/* I'm not releasing ftp support on cygwin */
+	b[i++] = uc_copy("/usr/share/fonts/");
+	b[i++] = uc_copy("/usr/share/X11/fonts/");
 	b[i++] = uc_copy("ftp://ctan.org/pub/tex-archive/fonts/");
-#endif
 	b[i++] = NULL;
 	GFileChooserSetBookmarks(b);
     } else {
