--- origsrc/fontforge-20170731/fontforgeexe/uiutil.c	2017-07-30 16:49:01.000000000 -0500
+++ src/fontforge-20170731/fontforgeexe/uiutil.c	2018-02-27 17:04:46.661432200 -0600
@@ -38,8 +38,7 @@ extern GBox _ggadget_Default_Box;
 #define MAIN_FOREGROUND (_ggadget_Default_Box.main_foreground)
 
 #if __CygWin
-extern void cygwin_conv_to_full_posix_path(const char *win,char *unx);
-extern void cygwin_conv_to_full_win32_path(const char *unx,char *win);
+#include <sys/cygwin.h>
 #endif
 
 static char browser[1025];
@@ -202,7 +201,7 @@ void help(char *file) {
 	#if __CygWin
 	if( strncmp(p_uri, "http:", 5) != 0 ){
 	    char*  temp = (char*) malloc(1024);
-	    cygwin_conv_to_full_win32_path(p_uri, temp);
+	    cygwin_conv_path(CCP_POSIX_TO_WIN_A, p_uri, temp, 1024);
 	    free(p_uri);
 	    p_uri = temp;
 	}
@@ -259,9 +258,9 @@ return;
     else
 #endif
     if ( strstr(fullspec,":/")==NULL ) {
-	if ( (temp=malloc(strlen(fullspec)+strlen("file:")+20))==NULL )
+	if ( (temp=malloc(strlen(fullspec)+strlen("file://")+20))==NULL )
 	    return;
-	sprintf(temp,"file:%s",fullspec);
+	sprintf(temp,"file://%s",fullspec);
 	strncpy(fullspec,temp,sizeof(fullspec));
     fullspec[sizeof(fullspec)-1] = '\0';
 	free(temp);
