--- origsrc/fontforge-20170731/configure.ac	2017-07-30 16:49:01.000000000 -0500
+++ src/fontforge-20170731/configure.ac	2018-03-15 12:07:50.410745800 -0500
@@ -511,7 +511,7 @@ case $host in
 	  my_lib_ldflags="${my_lib_ldflags} -no-undefined -Wl,--export-all-symbols"
       ;;
    *cygwin*)
-	  my_lib_ldflags="${my_lib_ldflags} -Wl,-no-undefined -Wl,--export-all-symbols"
+	  my_lib_ldflags="${my_lib_ldflags} -no-undefined -Wl,--export-all-symbols"
       ;;
    *) saved_LDFLAGS=$LDFLAGS
       AC_MSG_CHECKING([whether the linker supports --no-undefined])
